env0/data_teams.go

```diff
@@ -0,0 +1,48 @@
+package env0
+
+import (
+	"context"
+
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataTeams() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataTeamsRead,
+
+		Schema: map[string]*schema.Schema{
+			"names": {
```

Comment:
could have added "ids" as well. wasn't sure it will be useful.

Reply:
not sure it will help, but we can always add it in the future

Reply:
not sure it will help, but we can always add it in the future